src/generate/templates/header.rs

```diff
@@ -9,6 +9,13 @@ pub struct OwnedHandle<I: gll::runtime::Input, T: ?Sized> {
 }
 
 impl<I: gll::runtime::Input, T: ?Sized> OwnedHandle<I, T> {
+    pub fn new(forest_and_node: gll::runtime::OwnedParseForestAndNode<_P, I>) -> Self {
```

Comment:
Bad idea, this is a type that should only be created by "privileged code", and be otherwise opaque externally.

Reply:
Alright, the bad commit has been removed from history 💥